src/iconsManifest/supportedExtensions.ts

```diff
@@ -906,6 +906,11 @@ export const extensions: IFileCollection = {
       filenamesGlob: ['capacitor.config'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'capnp',
+      extensions: ['capnp'],
```

Comment:
I see some VSCode extensions register the `capnp` or `capnproto` language ID. It would be better to use those instead of the file extension, so the icon shows in the language picker too.

Personally I don’t mind having both `extensions` and `languages` (that adds support for people who don’t have the accompanying VSCode extension installed), but IIRC I’ve seen in other code reviews that it’s preferred to specify just the languages.

Reply:
The rationale here is that declared extensions have more priority than the ones declared by the extensions via language id. So if we declare, say, extension `.aaa` for language `A` and then for some reason the extension for language `A` changes to `.bbb`, we would have to change it ourselves, meanwhile if we are just relying on language ids, we move that reponsibility to the language id extensions. 

On the other hand, we don't show icons if the extensions supporting those language ids are not installed. That's the trade-off.

Reply:
@robertohuertasm So, is it better to add both the language and the extension or?

Reply:
We usually prefer to use the language id if available, not both.

Reply:
Hi @robertohuertasm, @remcohaszing I was wondering about the adjustments I made based on your feedback. Is everything in order, or do any extra modifications need to be made? Thank you in advance! 😁